src/languages/bash.coffee

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+
+  name: "Bash"
+  namespace: "bash"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "Shell script"
```

Comment:
Should this be `Shell Script` (not capitalization)?
See https://github.com/atom/language-shellscript/blob/master/grammars/shell-unix-bash.cson#L2